sapi/cli/php_cli.c

```diff
@@ -1304,7 +1312,8 @@ int main(int argc, char *argv[])
 	sapi_module->ini_entries = php_ini_builder_finish(&ini_builder);
 
 	/* startup after we get the above ini override se we get things right */
-	if (sapi_module->startup(sapi_module) == FAILURE) {
+	if (sapi_module->startup(sapi_module) == FAILURE
+			|| sapi_module->child_startup(sapi_module) == FAILURE) {
```

Comment:
I think there's not much point to force SAPI's to call child startup if they don't create any children.

Reply:
It would be better to let SAPI code to call it during startup if no child_startup defined.

Reply:
Agreed, but the separation of startup/child_startup happens to be useful even for these SAPIs in the context of opcache.preload